### ISSUE TITLE
There is no option is available to on the Live share accessibility feature.: A11y_VS Code_Funtionality

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#240647]
### GitHub Tags
#A11yTCS; #A11yFuntional;#Functionality; #Win32; #DesktopApp; #Visual Studio Code Client; #A11yMAS;#IRT:237009

### Environment Details:
Application Name: VS Code 
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Open VS Code.
2. Press CTRL+, to open the settings.
3. Search for live share in the search.
4. Tab till visual studio live share.
5. Tab till Accessibility feature enabled option.
6. Observe that there is no option is available to on the Live share accessibility feature.

### Actual Result:
there is no option is available to on the Live share accessibility feature.

### Expected Results:
On option should be available for the Live share accessibility feature.

### User Impact:
Users will not be able to turn on the live share accessibility feature.

### Attachment:

![Image](https://github.com/user-attachments/assets/84a5d009-eece-4ce2-b55a-1ab9d37b6b27)

**Comment by meganrogge** [comment]
this is a bug with the liveshare extension, not vs code. pls create an issue for them https://github.com/Microsoft/live-share
